### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by pasqualespica**

### CODE REVIEW

**Comment by cloudify** [comment]
@pasqualespica please rebase from `master`

**Comment by cloudify** [comment]
@pasqualedevita looks like the rebase didn't work out

**Comment by pasqualespica** [comment]
> @pasqualedevita looks like the rebase didn't work out
yeap, i don't know why, try it now.
You mentioned another Pasquale : )

File: README.md [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
 
-Use this template when creating new typescript projects.
+# io-pay
+- [io-pay](#io-pay)
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  
+Is a [WISP](https://docs.italia.it/italia/pagopa/pagopa-specifichepagamenti-docs/it/stabile/_docs/SANP_2.2_Sez2_Cap06_ComponentiTecnicheNodo.html#componente-wisp) replacement project and it will be the first web interface to allow guest (_citizen_) payment of [pagoPa](https://www.pagopa.gov.it/) system.
+
+From now wont be true _"pagoPA non è un sito dove pagare"_ it'll be too 🚀
```
Comment (cloudify) [inline README.md:9]: I guess `wont` to be a typo? I understand the essence but not exactly how you want to say it

File: README.md [hunk 2]
```diff
@@ -1,3 +1,25 @@
...
+
+## Prerequisites
+
+## Usage
+
+From one console typing :
```
Comment (cloudify) [inline README.md:15]:
```suggestion
Run the following commands in your console:
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,25 @@
...
+```sh
+yarn install
```
Comment (cloudify) [inline README.md:17]:
```suggestion
$ yarn install
```

File: README.md [hunk 4]
```diff
@@ -1,3 +1,25 @@
...
+yarn start
```
Comment (cloudify) [inline README.md:18]:
```suggestion
$ yarn start
```

File: README.md [hunk 5]
```diff
@@ -1,3 +1,25 @@
...
+  - [Prerequisites](#prerequisites)
+  - [Usage](#usage)
+  
+Is a [WISP](https://docs.italia.it/italia/pagopa/pagopa-specifichepagamenti-docs/it/stabile/_docs/SANP_2.2_Sez2_Cap06_ComponentiTecnicheNodo.html#componente-wisp) replacement project and it will be the first web interface to allow guest (_citizen_) payment of [pagoPa](https://www.pagopa.gov.it/) system.
+
+From now wont be true _"pagoPA non è un sito dove pagare"_ it'll be too 🚀
+
+## Prerequisites
+
+## Usage
```
Comment (cloudify) [inline README.md:13]:
```suggestion
## Running
```

File: README.md [hunk 6]
```diff
@@ -1,3 +1,25 @@
...
+
+From one console typing :
+```sh
+yarn install
+yarn start
+```
+
+then open browser [here](http://localhost:1234/index.html?p=12345)
+
+it's all rights you will see something like that 👍
```
Comment (cloudify) [inline README.md:23]:
```
You should now see something like the following
```

**Review by cloudify — CHANGES_REQUESTED** [review]

**Comment by cloudify** [comment]
@pasqualespica much better now, some checks are failing too

File: README.md (hunk 1 above)
Comment (pasqualespica) [reply README.md:9]: It's typo, but the sentence should be removed too,  it isn't true. `io-pay` at the moment it will be a WISP replacement

File: README.md (hunk 2 above)
Comment (pasqualespica) [reply README.md:15]: ok

File: README.md (hunk 3 above)
Comment (pasqualespica) [reply README.md:17]: why ? in this way you can copy/paste the command from `README`.
see https://github.com/pagopa/io-pagopa-node-mock for example

File: README.md (hunk 5 above)
Comment (pasqualespica) [reply README.md:13]: ok

File: README.md (hunk 6 above)
Comment (pasqualespica) [reply README.md:23]: ok

File: README.md (hunk 4 above)
Comment (pasqualespica) [reply README.md:18]: like above

File: README.md (hunk 3 above)
Comment (cloudify) [reply README.md:17]: We used to have the convention of prefixing the dollar sign for commands to be entered in a console, I guess the convention got lost over time, it's not super important

**Comment by cloudify** [comment]
@pasqualespica could you please rebase as I've merger #5 that should fix the build error

**Comment by pasqualespica** [comment]
> @pasqualespica could you please rebase as I've merger #5 that should fix the build error

done

**Comment by cloudify** [comment]
@pasqualespica uhm something's off, I don't see you changes to the read me anymore

File: package.json [hunk 7]
```diff
@@ -1,40 +1,40 @@
 {
-  "name": "parceltest",
```
Comment (cloudify) [inline package.json:2]: The changes to this file look like just formatting, in that case can you keep the file as is? We'll make formatting changes in a different PR if needed

**Review by cloudify — CHANGES_REQUESTED** [review]

File: package.json (hunk 7 above)
Comment (pasqualespica) [reply package.json:2]: ok, it's my VS code `Tab Size` preferences set to `4 spaces`

Comment (pasqualespica) [reply package.json]: restored

**Review by cloudify — APPROVED** [review]

File: README.md [hunk 8]
```diff
@@ -1,3 +1,24 @@
...
+
+## Prerequisites
+
+- [yarn](https://classic.yarnpkg.com/en/docs/getting-started)
+## Running
+
+Run the following commands in your console :
+```sh
+yarn install
+yarn start
```
Comment (cloudify) [inline README.md:17]: @pasqualespica actually I just realized that we're missing the `start` script, can you add it please?
